scripts/add_crashid_to_queue.py

```diff
@@ -0,0 +1,13 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# See socorro/scripts/add_crashid_to_queue.py
+
+import sys
+
+from socorro.scripts import add_crashid_to_queue
+
+
+if __name__ == '__main__':
+    sys.exit(add_crashid_to_queue.main(sys.argv[1:]))
```

Comment:
This is a stub. The reason for making this a stub that calls another module is that this is not in an importable place, so it's much harder to write unittests for functionality in this stub. The easy thing to do is make this a stub and put the functionality in a place that's easier to access for tests.

We want to be testing scripts that are important to our flow so anything that makes that doable is good.